src/index.ts

```diff
@@ -21,4 +21,10 @@ app.get('/', (req, res) => {
   res.sendFile('views/index.html', { root: process.cwd() })
 })
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+app.use((err, req, res, next) => {
+  console.error(err)
+  res.status(500).send('Internal server error')
+})
```

Comment:
Can you remove here before we get conflict, because I also give details in this PR
https://github.com/fingerprintjs/dx-team-mock-for-proxy-integrations-e2e-tests/pull/15/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R24

Reply:
Done!